buildtools/publish.js

```diff
@@ -50,6 +51,8 @@ for (const folder of folders) {
     }
     if (config.published) {
       console.log('Publishing ' + config.widgetId);
+      config.lastUpdatedAt = execSync(`git log -1 --format=%cI ${packageFile}`, {encoding: 'utf8'})
```

Comment:
```suggestion
      config.lastUpdatedAt = execSync(`git log -1 --format=%cI ${packageFile}`, {cwd: rootDir, encoding: 'utf8'})
```

Reply:
Also line 31 is wrong
`if (!fs.statSync(path.join(rootDir, folder)).isDirectory()) {`

Idea was that this script can be executed from anywhere.